frontend/src/global_styles/content/_enterprise.sass

```diff
@@ -53,10 +28,14 @@
   align-items: center
 
 .upsale-colored
-  color: $spot-color-feedback-warning-dark
+  color: var(--enterprise-upsale-color) !important
+
+.upsale-colored-background
+  background: var(--enterprise-upsale-color) !important
+  color: white !important
 
 .upsale-border-colored
-  border-color: $spot-color-feedback-warning-dark
+  border-color: var(--enterprise-upsale-color)
```

Comment:
It looks like both of these classes are not used any more.

Reply:
The background one is used in the SaaS button